### ISSUE TITLE
Submenu widget icon color should be inverted when "Invert Color" option selected

### ISSUE THREAD

**Issue by eengert** [jurialmunkey/skin.arctic.horizon.2#119]
Can you invert the color of icons on the submenu widget buttons when the "Invert Colors" option is selected in settings?  See screenshot, which shows that the white icons are difficult to see against a light highlight color.

![Screen Shot 2022-06-09 at 2 48 49 PM](https://user-images.githubusercontent.com/33561812/172922285-51258a3f-9bac-411a-bc56-70963b7fa3cc.png)

**Comment by Goldenfreddy0703** [comment]
Oh I had that same issue too, I wanted this skin to match the Google tv interface but submenu icons have a highlighted color and I was thinking of of inverting them myself in photoshop.

**Comment by eengert** [comment]
I'm not sure if your skin has control over the Up Next popup window, but the button text for Play and Close in Up Next also isn't getting inverted and can't be seen against a light highlight color.

**Comment by eengert** [comment]
> Oh I had that same issue too, I wanted this skin to match the Google tv interface but submenu icons have a highlighted color and I was thinking of of inverting them myself in photoshop.

I don't think that would be the best solution because then they would be hard to see on the buttons that weren't in focus (which have a dark grey background).  I think this is something that can be fixed in the skin because I can see the icons being color-inverted in other parts of the skin such as the menu.

**Comment by Goldenfreddy0703** [comment]
> > Oh I had that same issue too, I wanted this skin to match the Google tv interface but submenu icons have a highlighted color and I was thinking of of inverting them myself in photoshop.
> 
> I don't think that would be the best solution because then they would be hard to see on the buttons that weren't in focus (which have a dark grey background).  I think this is something that can be fixed in the skin because I can see the icons being color-inverted in other parts of the skin such as the menu.

I can see your point and it's not really a good solution but it is a last resort if necessary and for the menu icons, I could just changed them using autowidget and edit the menus however I want. I've been working with icons for years when it comes to kodi skins and I like to make them look very professional. Some addons lack icons which is why I add them.

**Comment by jurialmunkey** [comment]
Okay 0.1.29 should fix submenus and addon browser icons when inverted.

However, I cannot reproduce the UpNext issue. To me the buttons are already inverted just like in the info dialog.

**Comment by Goldenfreddy0703** [comment]
Thank you

**Comment by Goldenfreddy0703** [comment]
Hey i just tried it and they look really great, Thank you @jurialmunkey

**Comment by eengert** [comment]
The submenu widget icons are now inverted, thanks.

This is what I see in the Up Next dialog:

![IMG_1168](https://user-images.githubusercontent.com/33561812/173079383-d95c2aff-29b5-44d0-891c-6d9745f19102.jpeg)

**Comment by jurialmunkey** [comment]
That's not Up Next addon. This is what the Up Next addon looks like in this skin:

![screenshot00108](https://user-images.githubusercontent.com/4618719/173080085-490f0c07-4a7a-4907-9e45-607ad1f0b384.png)

**Comment by Goldenfreddy0703** [comment]
@jurialmunkey im sorry but what addon is that and where can i find that skin theme for the dialog. I've been trying to make it look like that but i had no luck.

**Comment by Goldenfreddy0703** [comment]
Actually nevermind, i found it but thank you so much for introducing me to the Up Next addon with your skin. Now i can fix that theming bug with other addons.

**Comment by eengert** [comment]
> That's not Up Next addon.

My mistake.  Closing the issue.  Thanks.

**Comment by xyzfre** [comment]
> @jurialmunkey im sorry but what addon is that and where can i find that skin theme for the dialog. I've been trying to make it look like that but i had no luck.

If you don't mind me asking does this Up Next add-on work with other plug-ins or was this really made for people who use library with Kodi That pop-up looks very nice

**Comment by jurialmunkey** [comment]
> > @jurialmunkey im sorry but what addon is that and where can i find that skin theme for the dialog. I've been trying to make it look like that but i had no luck.
> 
> If you don't mind me asking does this Up Next add-on work with other plug-ins or was this really made for people who use library with Kodi ,That pop-up looks very nice

It works particular well with the Netflix plugin. It works well with anything that creates a playlist queue (which also tmdbhelper players can be configured to do as long as the player uses setresolvedurl rather than a scripted source scraper that hacks xbmc.player)

 works with any plugin thar makes a playlist queue.

**Comment by Goldenfreddy0703** [comment]
> > @jurialmunkey im sorry but what addon is that and where can i find that skin theme for the dialog. I've been trying to make it look like that but i had no luck.
> 
> If you don't mind me asking does this Up Next add-on work with other plug-ins or was this really made for people who use library with Kodi ,That pop-up looks very nice

The popups do look very nice, i was trying to make it a theme for seren or kaito in the future. Im having trouble with that tho but again very nice themes you put in artic horizen 2.
